### PR TITLE
sized the navbar down and used the icons from figma

### PR DESCRIPTION
**Pull request by Draqode**
@abiona01 find an issue for it 😂😂😂😂

### CODE REVIEW

**Comment by abiona01** [comment]
Create an issue for it.

File: client/src/components/mainside/Mainside.jsx [hunk 1]
```diff
@@ -1,34 +1,18 @@
...
-
     <MainContainer>
       <Goal>
-        <SimpleModal />
-        <Deletemodal />
-        <BasicModal/>
-      </Goal>
-      <Goal primary>
-    {/* <Report /> */}
-        <Notification />
```
Comment (abiona01) [inline client/src/components/mainside/Mainside.jsx:28]: I'm highlighting for further review.

**Review by abiona01 — CHANGES_REQUESTED** [review]

**Review by abiona01 — APPROVED** [review]

**Comment by Asylcreek** [comment]
So, there are conflicts in this PR, please pull from the staging branch, resolve them conflicts and make the PR again

**Comment by Draqode** [comment]
fixes #500  
@abiona01

**Review by Asylcreek — CHANGES_REQUESTED** [review]
Please move the styles in Navbar.jsx into a separate file

**Comment by Draqode** [comment]
i didn't work on it but i'll do so now

**Pull request merged**